packages/client/hmi-client/src/components/sidebar-panel/simulation-result-sidebar-panel.vue

```diff
@@ -0,0 +1,33 @@
+<script setup lang="ts">
+import { Project } from '@/types/Project';
+import { useRouter } from 'vue-router';
+
+defineProps<{
+	project: Project;
+}>();
+
+const router = useRouter();
+const openSimulationResult = (runId: number) => {
+	router.push({ params: { simulationRunId: runId } });
+};
+</script>
+
+<template>
+	<ul>
+		<li
+			v-for="runId of project.assets.simulation_runs"
+			:key="runId"
+			class="selectable-item"
+			@click="openSimulationResult(runId)"
+		>
+			{{ runId }}
+		</li>
+	</ul>
+</template>
+
+<style>
+.selectable-item:hover {
+	background-color: #ddd;
```

Comment:
We use [Uncharted Design Tokens](https://github.com/unchartedsoftware/design-tokens/blob/main/build/scss/_uncharted-variables.scss#L17)